src/metabase/api/user.clj

```diff
@@ -299,17 +297,17 @@
     (api/checkp (not (db/exists? User, :%lower.email (if email (u/lower-case-en email) email), :id [:not= id]))
                 "email" (tru "Email address already associated to another user."))
     (db/transaction
-     ;; only user or self can update user info
+     ;; only superuser or self can update user info
      ;; implicitly prevent group manager from updating users' info
      (when (or (= id api/*current-user-id*)
                api/*is-superuser?*)
        (api/check-500
         (db/update! User id (u/select-keys-when body
-                                                :present (into #{:locale} (when api/*is-superuser?* [:login_attributes]))
-                                                :non-nil (set (concat [:first_name :last_name :email]
-                                                                      (when api/*is-superuser?*
-                                                                        [:is_superuser]))))))
-       (maybe-update-user-personal-collection-name! user-before-update first_name last_name))
+                              :present (cond-> #{:first_name :last_name :locale}
+                                         api/*is-superuser?* (conj :login_attributes))
+                              :non-nil (cond-> #{:email}
+                                         api/*is-superuser?* (conj :is_superuser)))))
```

Comment:
I think this not the standard indentation.

Reply:
I think it's ok. Other u/select-keys-when instances in the codebase have the same indentation.